src/appleseed/foundation/platform/path.cpp

```diff
@@ -133,6 +135,49 @@ const char* get_executable_directory()
     return path;
 }
 
+const char* get_home_directory()
+{
+    static char path[FOUNDATION_MAX_PATH_LENGTH + 1];
+    static bool path_initialized = false;
+
+    if (!path_initialized)
+    {
+#if defined _WIN32
+
+        return 0;
+
++#elif defined __APPLE__
+
+        return 0;
+
+#elif defined __linux__
```

Comment:
Same thing, I'm pretty sure we want to have the same behavior on FreeBSD, as discussed with danfe.